docs/src/components/Layout/Footer.tsx

```diff
@@ -1,27 +1,71 @@
-import { Icon, Link, Text, Flex, useTheme } from '@aws-amplify/ui-react';
+import {
+  Icon,
+  Link,
+  Text,
+  Flex,
+  View,
+  useTheme,
+  Button,
+} from '@aws-amplify/ui-react';
+
+import * as links from '@/data/links';
+import * as icons from '@/data/icon';
+import { Logo } from '@/components/Logo';
 
 export const Footer = () => {
   const { tokens } = useTheme();
   return (
-    <footer className="docs-footer">
-      <Flex direction="row">
-        <Text color={`${tokens.colors.font.secondary}`}>
-          AWS Amplify open source, documentation and community are supported by
-          Amazon Web Services © 2021, Amazon Web Services, Inc. and its
-          affiliates. All rights reserved. View the site terms and privacy
-          policy.
-        </Text>
-        <Link
+    <Flex as="footer" direction="column" className="docs-footer">
+      <Flex direction="row" justifyContent="center">
+        <Logo />
+      </Flex>
+      <View textAlign="center">
+        <Button
+          as="a"
```

Comment:
It's pretty awesome how well the polymorphism seems to be working! Seems pretty straightforward to get a button-like link here.

Reply:
Yup!